week-1-ts-ecosystem/w01e01-typescript-basics/src/controller.ts

```diff
@@ -0,0 +1,11 @@
+import { ADULT_AGE, User } from './data'
+
+export function hasAddress(user: User): boolean {
+    return Boolean(user.address)
+  }
+  
+function hasGivenAge(requiredAge: number): Function {
+    return (user: User): boolean => user.age >= requiredAge
+  }
+  
+export const isAdult = hasGivenAge(ADULT_AGE)
```

Comment:
To nie do końca jest `controller` - bardziej widzę to jako `user.validators`

Reply:
Zmieniłem nazwę, faktycznie bardziej pasuje.